topics:
  p_cloud: "/pcloud"
  odom: "/syncOdom"
  laser: "/scan"

grid:
  x: 50
  y: 50
  dim: 0.1

params:
  p_thresh: 0.8
  p_occ: 0.65
  p_free: 0.35
  z_min: -0.5
  z_max: 0.5
  z_floor: -0.8

interface:
  display: False
  rviz: True
